modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -1052,42 +954,32 @@ test(
 	},
 
 	async ({apiHelpers, page}) => {
-		const pageTitle1 = 'My Page 1';
-		const sitePage1 = await createSitePage({
-			apiHelpers,
-			pageTitle: pageTitle1,
-		});
-		const pageTitle2 = 'My Page 2';
-		const sitePage2 = await createSitePage({
-			apiHelpers,
-			pageTitle: pageTitle2,
-		});
+		const pageName2 = 'My Page 2';
```

Comment:
@rafaellajordao please rename. Bare in mind that only makes sense to have `pageName2`  if you have a  `pageName1` :wink: